src/mscorlib/src/System/Collections/Generic/KeyValuePair.cs

```diff
@@ -62,5 +62,12 @@ public struct KeyValuePair<TKey, TValue> {
             s.Append(']');
             return StringBuilderCache.GetStringAndRelease(s);
         }
+
+        //[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
So it won't be filtered in Intelisense once/if this shows up in .NET Framework ... it does not sound like a bit problem to me.